src/application/services/useNote.ts

```diff
@@ -265,6 +278,35 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     parentNote.value = undefined;
   }
 
+  /**
+   * Reform the received note parents from api into presentation format.
+   * @returns An array of Note objects representing the formatted note parents.
+   * @throws {Error} If the note id is not defined.
+   */
```

Comment:
I dont really understand what this method is doing. Could you please improve jsdoc to make it more clear? Why API format is not suitable?